server/base/src/main/java/org/apache/accumulo/server/util/DumpZookeeper.java

```diff
@@ -129,12 +129,12 @@ private static void write(PrintStream out, int indent, String fmt, Object... arg
     for (int i = 0; i < indent; i++) {
       out.print("  ");
     }
-    out.println(String.format(fmt, args));
+    out.printf(fmt, args);
```

Comment:
In this instance, it might actually make sense to change `fmt` to `fmt + %n` rather than update each caller separately. The idea here is to minimize the potential for error and to ensure consistency by doing the work in fewer places:

```suggestion
    out.printf(fmt + "%n", args);
```

Of course, changing this would mean reverting the changes to the other lines in this file.